src/components/PersonRow/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <td>
+      <Link
+        to={{
+          pathname: `./${person.slug}`,
```

Comment:
DRY. Destruct your props and use short names.

Reply:
Denied.
It's only style of coding, moreover I use it only one time and don't need to reuse